app.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'lib/database_connection'
+require_relative 'lib/item_repository'
+require_relative 'lib/order_repository'
+
+class Application
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
+  def run
```

Comment:
Code looks really well structured, it would be refactored into methods to reduce the size of the run method.